src/components/Filters/Filters.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import styles from '../../common/commonOptionButton.module.css';
+import filterStyles from './Filters.module.css';
+
+const filterOptions = ['Animals', 'Health', 'Education', 'Environment'];
+
+function Filters() {
+   const [selectedFilters, setSelectedFilters] = useState<string[]>([]);
+   const [isOpen, setIsOpen] = useState<boolean>(false);
+
+   const toggleIsOpen = () => {
+      setIsOpen(!isOpen);
+   };
+
+   const handleFilterClick = (filter: string) => {
+      if (selectedFilters.includes(filter)) {
+         setSelectedFilters(selectedFilters.filter((f) => f !== filter));
+      } else {
+         setSelectedFilters([...selectedFilters, filter]);
+      }
+   };
+
+   return (
+      <div className={filterStyles.filters}>
+         <button
+            className={`${styles.button} ${isOpen ? styles.open : null}`}
+            onClick={toggleIsOpen}>
+            Filters: {selectedFilters.join(', ')} {isOpen ? '▲' : '▼'}
+         </button>
```

Comment:
What will happen if in the future you will have more filter options? Is that how you would want the button and the text to behave? What are the solutions and which one would you prefer?

Reply:
not ideal but enough for now.